Remove references to sun packages

Using `com.sun.*` packages is considered bad practice -  usage of `com.sun.xml.internal.ws.api` in `I18.java` breaks compilation under javac 1.8.0_302
```
src/yass/I18.java:21: error: package com.sun.xml.internal.ws.api does not exist
import com.sun.xml.internal.ws.api.ResourceLoader;
                                  ^
```

--- a/src/yass/I18.java
+++ b/src/yass/I18.java
@@ -18,8 +18,6 @@
 
 package yass;
 
-import com.sun.xml.internal.ws.api.ResourceLoader;
-
 import javax.imageio.ImageIO;
 import javax.swing.*;
 import java.awt.*;

